### PR TITLE
fix(core): fix minimap positioning issue when no results are present in table

### PR DESCRIPTION
**Pull request by dhirendra777**
affects: @medly-components/core

### PR Checklist

Please check if your PR fulfills the following requirements:

-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [ ] Docs have been added / updated (for bug fixes / features)

### PR Type

What kind of change does this PR introduce?

-   [x] Bugfix
-   [ ] Feature
-   [ ] Code style update (formatting, local variables)
-   [ ] Refactoring (no functional changes, no api changes)
-   [ ] Build related changes
-   [ ] CI related changes
-   [ ] Performance improves
-   [ ] Adding missing tests
-   [ ] Documentation content changes
-   [ ] Other... Please describe:

### Does this PR introduce a breaking change?

-   [ ] Yes
-   [x] No

### Additional context

Add any other context or screenshots about the feature pr here.

### CODE REVIEW

File: packages/core/src/components/Table/Body/Body.tsx [hunk 1]
```diff
@@ -12,6 +12,16 @@ const Body: React.FC<Props> = React.memo(props => {
             TableComponentsCommonPropsContext
         ),
         { selectedRowIds, onRowSelection, onGroupedRowSelection, setUniqueIds, ...restProps } = props,
+        /* since minimap is positioned sticky with respect to the tbody, tbody should have full table width otherwise minimap positioning fails */
+        getNoResultRowColumnTemplate = useCallback(() => {
+            let scrollWidth = 0,
```
Comment (gmukul01) [inline packages/core/src/components/Table/Body/Body.tsx:17]: You can do something like this ---> `scrollWidth = tableRef.current?.scrollWidth ?? 0;`

**Review by gmukul01 — CHANGES_REQUESTED** [review]

File: packages/core/src/components/Table/Body/Body.tsx [hunk 2]
```diff
@@ -8,10 +9,12 @@ import { NoResultCell, NoResultRow } from './Row/Row.styled';
 import { Props } from './types';
 
 const Body: React.FC<Props> = React.memo(props => {
-    const { data, groupBy, rowIdentifier, showRowWithCardStyle, noResultRow, tableRef, withMinimap, columns } = useContext(
+    const { data, groupBy, rowIdentifier, showRowWithCardStyle, noResultRow, tableRef, withMinimap, columns, size } = useContext(
             TableComponentsCommonPropsContext
         ),
         { selectedRowIds, onRowSelection, onGroupedRowSelection, setUniqueIds, ...restProps } = props,
+        /* since minimap is positioned sticky with respect to the tbody, tbody should have full table width otherwise minimap positioning fails */
+        getTableVisibleWidth = useCallback(() => (tableRef.current ? tableRef.current.clientWidth : 0), []),
```
Comment (gmukul01) [inline packages/core/src/components/Table/Body/Body.tsx:17]: You can simply use a variable over here ->>> tableClientWidth = tableRef.current?.clientWidth ?? 0;

**Review by gmukul01 — CHANGES_REQUESTED** [review]

File: packages/core/src/components/Table/Body/Body.tsx (hunk 2 above)
Comment (dhirendra777) [reply packages/core/src/components/Table/Body/Body.tsx:17]: got it

**Review by gmukul01 — APPROVED** [review]
LGTM

**Pull request merged**